### DISCUSSION TITLE
Poll: Help Us Choose Our New Name! 🚀

### DISCUSSION THREAD

**Discussion by tjbck** [open-webui/open-webui#720] [Polls]
Thanks for all the name suggestions! We narrowed down the candidates based on the SEO results, and the availability of GitHub organisation usernames, Help us choose our new name:

1. L3x: Local Large Language (Model) Experience.
2. Dog3: Distributed Open GenAI 3xperience.
3. One-webui: Unity & integration of multiple LLMs.
4. Open-webui: Openness & flexibility for various LLMs.

Your input shapes our future!

**Comment by explorigin** [comment]
With DOG3, you could go recursive...DOG3 Open GenAI Experience!

**Reply by mak448a** [reply]
These comments made me change my vote

**Comment by functorism** [comment]
Dog3 is so stupid that I love it :partying_face:

**Comment by bolavefasfas** [comment]
How about llm-webui or something else it should be catchy.

**Comment by unlikezy** [comment]
open-webui is good enough.
